raiden/network/transport/matrix/transport.py

```diff
@@ -1338,33 +1341,64 @@ def _send_with_retry(self, queue: MessagesQueue) -> None:
 
     def _send_raw(self, receiver_address: Address, data: str) -> None:
         assert self._raiden_service is not None, "_raiden_service not set"
-        room = self._get_room_for_address(receiver_address, require_online_peer=True)
 
+        # Try sending using webRTC
+        if self._web_rtc_manager.has_ready_channel(receiver_address):
+            self.log.debug(
+                "Send raw using webRTC message",
+                receiver=to_checksum_address(receiver_address),
+                data=data.replace("\n", "\\n"),
+            )
+            rtc_partner = self._web_rtc_manager.address_to_rtc_partners[receiver_address]
+            send_rtc_message(rtc_partner, data, self._raiden_service.address)
+            return
+
+        # Try sending using toDevice message
+        capabilities = self._address_mgr.get_address_capabilities(receiver_address)
+        if self._capability_usable(Capabilities.TODEVICE, capabilities):
+            self.log.debug(
+                "Send raw using toDevice message",
+                receiver=to_checksum_address(receiver_address),
+                data=data.replace("\n", "\\n"),
+            )
+            online_userids = {
+                user_id
+                for user_id in self._address_mgr.get_userids_for_address(receiver_address)
+                if self._address_mgr.get_userid_presence(user_id) in USER_PRESENCE_REACHABLE_STATES
+            }
+            if not online_userids:
+                self.log.debug("No online user_ids", online_userids=online_userids)
+                return
+
+            body = {
+                user_id: {"*": {"msgtype": "m.text", "body": data}} for user_id in online_userids
+            }
```

Comment:
just a note I saw here, there should never be multiple user_ids online for the same address as it would mean that the same address is used by multiple raiden clients at the same time